tests/test_working_copy_sqlserver.py

```diff
@@ -11,6 +12,7 @@
 H = pytest.helpers.helpers()
 
 
+@pytest.mark.xfail(is_linux, reason="pyodbc is not yet included in linux build")
```

Comment:
for future reference, you can actually do this [once at the module level](https://docs.pytest.org/en/latest/example/markers.html#marking-whole-classes-or-modules):

```python
# xfail all tests in this module if we're on Linux
pytestmark = pytest.mark.xfail(is_linux, reason="pyodbc is not yet included in linux build"))
```